felix/fv/etcd_restart_test.go

```diff
@@ -60,7 +58,7 @@ var _ = Context("etcd connection interruption", func() {
 		// Wait until the tunl0 device appears; it is created when felix inserts the ipip module
 		// into the kernel.
 		Eventually(func() error {
-			links, err := netlink.LinkList()
+			links, err := utils.LinkList()
```

Comment:
`utils` is best avoided, it ends up being a catchall and you could have 100 utils packages with different random stuff in them.

Can we emphasize the retry somehow; important for the user to know that this is not he same as the basic version.

`netlinkretry.LinkList()`
`netlinkutils.LinkListRetryEINTR()`